src/bootstrap/lib.rs

```diff
@@ -170,6 +170,8 @@ use crate::cache::{Interned, INTERNER};
 pub use crate::config::Config;
 pub use crate::flags::Subcommand;
 
+use libc;
```

Comment:
I think this shouldn't be necessary? Did you see compilation failures without it?

Reply:
Oh it does work without the `use`. I assumed it would be needed since it's not declared anywhere in the module. Is it included in the prelude just for `rustbuild`?

Reply:
libc is an external crate and since the 2018 edition, those are available without explicit use statements (well, extern crate).

Reply:
Ah right, of course. It slipped my mind that `use` is used to alias paths